### ISSUE TITLE
Tiles - Styled Components

### ISSUE THREAD

**Issue by gbisbjerg** [CS-Social-Good-CalPoly/Vera#53]
Problem
---

Throughout the site we repeatedly use tiles that have extremely similar style. To reuse code as well as make possible quick and easy changes to all of the tiles in the future, styled components should be used. 

Do This
---
Modify existing tiles (the square ones, *not* the expanded tile mentioned below) to use shared styled components when possible, and then use those shared styles to complete remaining tiles.

Start by creating a tile styled component and implementing it for the `StoryTile` component. This should contain the css for the shape of the tile, shadow, the banner image, and any other shared items.

Focus on implementing the newly generated styled component to the `ResourcePageTile` and checking that no visual changes occurred. 

For the `IndividualResourceTile`, we already have the expanded version coded. However, we're going to want a collapsed version too! Right now, the expanded version is inside the file for the `IndividualResourceTile`. We'll want to put the expanded code into it's own component. Then, we'll want a new component for the collapsed/minimized version of the component. The `IndividualResourceTile` component will use state to track whether the tile should be collapsed or expanded (the state should change upon clicking either the collapsed tile itself *or* the `x` button for the expanded tile). 

To summarize:
- Rename the `IndividualResourceTile` to `IndividualResourceTileExpanded`
- Create a collapsed version named `IndividualResourceTileCollapsed`. This one will use the styled components that it has in common with the other tile components we have.
- Through the use of a on click state change, the component should swap between each other. 

<img width="156" alt="Screen Shot 2021-04-19 at 10 03 33 PM" src="https://user-images.githubusercontent.com/23414761/115340184-17e65900-a15b-11eb-826c-55efe8c40297.png">
<img width="635" alt="Screen Shot 2021-04-19 at 10 03 21 PM" src="https://user-images.githubusercontent.com/23414761/115340161-1026b480-a15b-11eb-94b9-ef05943a61ac.png">
